week1/sui-bank/sources/tests/exercise_tests.move

```diff
@@ -0,0 +1,50 @@
+module sui_bank::exercise_tests { 
+
+  use sui::test_utils::assert_eq;
+  use sui::coin::{mint_for_testing};
+  use sui::test_scenario::{Self as test, next_tx, ctx};
+
+  use sui_bank::solution::{Self, Bank};
+
+  const ADMIN: address = @0xBEEF;
+  const ALICE: address =  @0x1337;
+
+  #[test]
+  fun test_deposit() {
+    let scenario = test::begin(@0x1);
```

Comment:
in docs.sui.io

let scenario_val = test_scenario::begin(admin);
let scenario = &mut scenario_val;